code/ATMOSPHERICS/components/unary/vent_scrubber.dm

```diff
@@ -60,10 +62,14 @@
 	if(!istype(T))
 		return
 
-	if(!powered())
-		scrubber_icon += "off"
+	if(welded)
```

Comment:
scrubbers are now weldable. Mobs can crawl through the waste pipes as well as the normal ones, so this allows welding scrubbers to keep creatures out